tests/test_standard_tags.py

```diff
@@ -80,19 +80,71 @@ def setup_url_with_sensitive_query_string(self):
             ),
         ]
 
+    # when tracer is updated, add (for exemple)
+    # @irrelevant(context.library >= "python@1.20.0", reason="python released the new version at 1.20.0")
     def test_url_with_sensitive_query_string(self):
```

Comment:
two tests have the same name now ?? line 85 and 118. I think you mean this:
```suggestion
    def test_url_with_sensitive_query_string_legacy(self):
```